### PR TITLE
Schema is marked as required

### PR DESCRIPTION
**Pull request by vjosadika**
Jira ticket: https://issues.cask.co/browse/PLUGIN-251

### CODE REVIEW

**Review by rmstar — COMMENTED** [review]
I don't think removing the Nullable annotation is sufficient here. You'll also need to add some validation to check that schema is not null.

File: src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerSourceConfig.java [hunk 1]
```diff
@@ -25,9 +25,9 @@
 import io.cdap.plugin.gcp.common.GCPReferenceSourceConfig;
 import io.cdap.plugin.gcp.spanner.common.SpannerUtil;
 
+import javax.annotation.Nullable;
```
Comment (rmstar) [inline src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerSourceConfig.java]: Why was this import reordered?

**Review by rmstar — COMMENTED** [review]
lgtm, 1 minor comment.

File: src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerSourceConfig.java (hunk 1 above)
Comment (vjosadika) [reply src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerSourceConfig.java]: Fixed

**Comment by rmstar** [comment]
Please squash commits.

**Review by CuriousVini — APPROVED** [review]

**Pull request merged**